circle2/deploy-java-service.sh

```diff
@@ -34,7 +34,7 @@ fi
 
 # Deploy to EKS if the service config exists in the microservice manifests
 [[ -d ${MANIFESTS_DIR} ]] || git clone --single-branch --branch ${MANIFESTS_BRANCH} https://${WOMPLY_CIRCLECI_SHARED_USER_GITHUB_ACCESS_TOKEN}@github.com/${MANIFESTS_GITHUB_REPO}.git ${MANIFESTS_DIR}
-if [[ -f "${MANIFESTS_DIR}/src/environments/${EKS_ENVIRONMENT}/image-tags/${CIRCLE_PROJECT_REPONAME}.yaml" ]]; then
+if [[ -f "${MANIFESTS_DIR}/src/environments/${EKS_ENVIRONMENT}/${CIRCLE_PROJECT_REPONAME}/image-tags.yaml" ]]; then
```

Comment:
`image-tag` please... it should be singular, now that we have only one such a file per subdirectory.